.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
   sign:
     needs: build
     runs-on: windows-latest # Code signing must run on a Windows agent for Authenticode signing (dll/exe)
-
+    if: github.repository == 'dotnet/MQTTnet'
     steps:
```

Comment:
maybe also not run it from a PR, to avoid supply chain attacks that produce signed packages